rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndParameterLookup.scala

```diff
@@ -501,6 +618,30 @@ class TestNodeAndParameterLookup extends Specification {
       compare("${node.properties[datacenter][Europe]}", ("datacenter", json) :: Nil) must beEqualTo("""{"France":"Paris"}""") // look, NO SPACES
     }
 
+    "correctly return the same string if interpretation is done on node" in {
+      compare("${node.properties[datacenter][Europe]|node}", ("datacenter", json) :: Nil) must beEqualTo("""${node.properties[datacenter][Europe]}""")
+    }
+
+    "correctly return the default string if value is missing on node" in {
+      compare("""${node.properties[datacenter][Europe]|default="some default"}""", ("missing_datacenter", json) :: Nil) must beEqualTo("""some default""")
+    }
+
+    "correctly return the replaced value if key is present on node" in {
+      compare("""${node.properties[datacenter][Europe]|default="some default"}""", ("datacenter", json) :: Nil) must beEqualTo("""{"France":"Paris"}""")
+    }
+
+    "correctly return the replaced value, two level deep, if first key is missing" in {
+      compare("""${node.properties[missing][key]|default= ${node.properties[datacenter][Europe]|default="some default"}}"""
+           , ("datacenter", json) :: Nil) must beEqualTo("""{"France":"Paris"}""")
+    }
+    "correctly return the default value, two level deep, if all keys are missing" in {
+      compare("""${node.properties[missing][key]|default= ${node.properties[missing_datacenter][Europe]|default="some default"}}"""
+           , ("datacenter", json) :: Nil) must beEqualTo("""some default""")
+    }
+    "correctly return the default value which is ${node.properties...}, if default has the 'node' optiopn" in {
+      compare("""${node.properties[missing][key]|default= ${node.properties[datacenter][Europe]|node}}"""
+           , ("datacenter", json) :: Nil) must beEqualTo("""${node.properties[datacenter][Europe]}""")
```

Comment:
Could you also add some tests with feature disabled, to check that it is working as expected